src/run_post_deploy_scripts.sh

```diff
@@ -17,17 +17,31 @@ done
 
 
 echo "Now check the common post deploy scripts and run them as well"
+echo "Check if post-deploy scripts are persisted from previous job"
+if [ -d /tmp/workspace/base-fork/post-deploy ]; then
 
-mkdir source
-pushd source
-git clone https://github.com/greenpeace/planet4-base-fork .
-popd
+    for file in /tmp/workspace/base-fork/post-deploy/*; do
```

Comment:
All this is repeated on line 39.
Since you persist it on the workspace, is there any case ever where it would not be there? (In which case you would need to git pull it? 
If there is no such case, then this should be simplified further. (Only once the same code, and no git pull at all).

Reply:
Yes, I left for safety there.
It will need changes on all NRO repos (persisting the repo) for this work. Or maybe there is another way I'm not aware of.

That way even if the NRO's env does not persist the repo code, will still clone it.

Reply:
With [this](https://github.com/greenpeace/planet4-circleci-config) I can push the changes to all nros circleCI configurations. The places where we depend on the code persisting from the previous steps are many in our configuration, so accomodating for that one case not having correct config.yml would not help a lot.

Reply:
I didn't know about that. I would remove the cloning code then.